docs/full-node/manage-a-terra-validator/set-up-validator.md

```diff
@@ -68,3 +68,21 @@ In general a validator needs to do three things well
 - Conduct on-chain operations such as voting on Governance proposals (using an Application Operator Key)
 
 Protecting and having a contingency backup plan for all your [keys](faq.md#what-are-the-different-types-of-keys) will help mitigate catastrophic hardware or software failures of the node.
+It is a good practice to test your backup plan on a testnet node in case of node failure.
+
+### 4.1 Restoring a validator
+
+A validator can be completely restored on a new Terra node given the following set of keys:
+- Consensus key, stored in `~/.terra/config/priv_validator.json`
+- The mnemonic to the validator wallet
```

Comment:
this is not really correct.

while you should be storing your validator & feeder mnemonics securely, they are not required to restore a validator node.
you just need the   **priv_validator_key.json** (not priv_validator.json).

the validator wallet does NOT need to be stored anywhere on the cloud, or on the machine itself. (and really shouldn't be).

for the feeder, you should only store the 'voter.json' file (and the decryption password you use) on cloud.

Reply:
That's a good point - the validator keys can be kept completely separate and any tx's can be issued on a separate machine